master/usage/removing-nodes.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Removing Nodes
+---
+
+With each Calico node there is a Node entry in the datastore and Workload
+Endpoints are associated with that node.  As nodes are replaced (with
+differently named nodes) or decomissioned, the datastore will accumulate
+node entries and possibly workload entries.  Currently there is not an
+automatic reaping of these old Node and Endpoint entries so it is necessary
```

Comment:
I'd probably do a more positive spin, by simply stating that if you decommission a node (which includes renaming an existing node) then it is necessary to explicitly delete the node configuration to release resources associated with that node.  This includes blah blah and blah. 

 You could go one step further and suggest building it into the cluster management system (whatever the term is)